src/templates/ssr/fabric/index.svelte

```diff
@@ -0,0 +1,154 @@
+<!-- https://polyfill.io/v3/polyfill.min.js?features=default -->
+<script lang="ts">
+	import { CLICK_MACRO, GAMVariable } from '$lib/gam';
+	import { isValidReplacedVariable } from '$lib/gam';
+	import AdvertisementLabel from '$templates/components/AdvertisementLabel.svelte';
+	import Pixel from '$templates/components/Pixel.svelte';
+
+	export let Trackingpixel: GAMVariable;
+	export let Researchpixel: GAMVariable;
+	export let Viewabilitypixel: GAMVariable;
+	export let ClickthroughUrl: GAMVariable;
+	export let thirdPartyJSTracking: GAMVariable;
+
+	export let Layer1BackgroundImage: GAMVariable;
+	export let Layer1BackgroundPosition: GAMVariable;
+	export let Layer2BackgroundImage: GAMVariable;
+	export let Layer2BackgroundPosition: GAMVariable;
+	export let Layer3BackgroundImage: GAMVariable;
+	export let Layer3BackgroundPosition: GAMVariable;
+
+	export let MobileLayer1BackgroundImage: GAMVariable;
+	export let MobileLayer1BackgroundPosition: GAMVariable;
+	export let MobileLayer2BackgroundImage: GAMVariable;
+	export let MobileLayer2BackgroundPosition: GAMVariable;
+	export let MobileLayer3BackgroundImage: GAMVariable;
+	export let MobileLayer3BackgroundPosition: GAMVariable;
+</script>
+
+<AdvertisementLabel />
+<div class="creative creative--fabric">
+	<a
+		id="linkDesktop"
+		class="blink gs-container link hide-until-tablet"
```

Comment:
we can merge the classes here that aren't reused for simplicity

Reply:
I don't really understand what do you mean by merge the classes?

Reply:
As in merge the separate classes and styles here into a single class and styles, unless there's an obvious need to reuse one. Looks like `gs-container` doesn't have any styles in this file either?

Reply:
Ah ok got it. Yes I noticed that now and added it.

Reply:
`blink`, `gs-container` and `link` are reused in 2 places in this file so not quite sure how to merge them if they are reused. `hide-until-tablet` is this for specific anchor tag.

Reply:
You're right, but all these classes/styles aren't doing very much I think it would be nicer to just a couple more semantic class names like a `link` class for the shared styles and `link-desktop` and `link-mobile` classes for the specific styles.

Reply:
Refactored with Jake on a call.